README.md

```diff
@@ -7,36 +7,72 @@
 
 ## Description
 
-A flexible front end to plate bases pipelines in Sequencescape
+A flexible front end to plate bases pipelines in Sequencescape.
 
 ## Initial Setup
 
-1. In Limber, connect to Sequencescape to configure required data:
-```shell
-  bundle exec rake config:generate
-```
+Steps must be followed in either the Sequencescape repository or this Limber
+repository, as indicated:
 
-2. In Sequencescape, configure all Limber required data:
-```shell
-  bundle exec rake limber:setup
-```
+1. In Limber, ensure the appropriate version of Ruby is installed. The command
+   here is for `rbenv` but you may want to use a different Ruby version manager.
+   If the command fails, you might need to check the list of available Ruby
+   versions using `rbenv install --list-all` and use the one that most closely
+   matches the contents of `.ruby-version`:
```

Comment:
I'm not sure we want to encourage developers to use different ruby versions.

Reply:
Sorry maybe this reads wrong.  When I tried to run `rbenv install` it told me that there was no Ruby version for `ruby-2.7.2`, which it picks up automatically from the `.ruby-version` file.  So I ran `rbenv install --list-all` and found `2.7.2` without the `ruby-` prefix.  After that installed with `rbenv install 2.7.2`, `rbenv` referred to it as `ruby-2.7.2` so either the `.ruby-version` file will need to be changed, or these instructions need to be here to explain what to do when `rbenv` cannot find `ruby-2.7.2`.

Reply:
Ahh I see... that makes sense. I've been updating most our apps to say just 2.7.2 in `.ruby-version` for precisely this reason, bit surprised that limber hasn't been updated yet.

Reply:
I do find it weird that rbenv is more permissive when switching ruby versions than installing them. I know installing is provided by a plugin, but would still expect them to behave the same.

Reply:
I've updated the `.ruby-version` file now and removed these additional confusing instructions.